website/docs/r/ip_space.html.markdown

```diff
@@ -0,0 +1,224 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_ip_space"
+sidebar_current: "docs-vcd-resource-ip-space"
+description: |-
+  Provides a resource to manage IP Spaces for IP address management needs. IP Spaces provide 
+  structured approach to allocating public and private IP addresses by preventing the use of 
+  overlapping IP addresses across organizations and organization VDCs.
+---
+
+# vcd\_ip\_space
+
+Provides a resource to manage IP Spaces for IP address management needs. IP Spaces provide
+structured approach to allocating public and private IP addresses by preventing the use of
+overlapping IP addresses across organizations and organization VDCs.
+
+IP Spaces require VCD 10.4.1+ with NSX-T.
+
+## Example Usage (Private)
```

Comment:
What about:

```suggestion
## Example Usage (Private IP Space)
```

Reply:
This was exactly what I was considering as well. Somehow I left it this way when looking at web page preview because it looked to me that in that view one can see the resource title and has enough context when looking at the right side.

Changed it now as it might not be the only view that people see :)


<img width="954" alt="image" src="https://github.com/vmware/terraform-provider-vcd/assets/15804230/ce554349-7d03-4097-a60c-bf760a1100e7">